modules/architecture-rhcos-updating-bootloader.adoc

```diff
@@ -0,0 +1,88 @@
+// Module included in the following assemblies:
+//
+// * architecture/architecture_rhcos.adoc
+
+[id="architecture-rhcos-updating-bootloader.adoc_{context}"]
+= Updating the bootloader using bootupd
+
+To update the bootloader using bootupd, you must either install bootupd on {op-system} machines manually, or provide a machine config with the enabled systemd unit. Unlike grubby or other bootloader tools, bootupd does not manage kernel space configuration such as passing kernel arguments.
+
+Once installed, you can manage bootupd remotely from the {product-title} cluster.
+
+[NOTE]
+====
+It is recommended that you use bootupd only on bare metal or virtualized hypervisor installations, such as for protection against the BootHole vulnerability.
+====
+
+.Manual install method
+You can manually install bootupd using the `bootctl` command-line tool.
+
+. Inspect the system status:
++
+[source,terminal]
+----
+# bootupctl status
+----
++
+.Example output
+[source,terminal]
+----
+Component EFI
+  Installed: grub2-efi-x64-1:2.04-31.fc33.x86_64,shim-x64-15-8.x86_64
+  Update: At latest version
+----
+
+[start=2]
+. Older {op-system} images that predate the existence of bootupd require an explicit adoption phase.
```

Comment:
Is there a more concrete way that you can put this?

Reply:
I've changed to:

> . {op-system} images created without `bootupd` installed on them require an explicit adoption phase.
> +
> If the system status is `Adoptable`, perform the adoption: